src/mmw/js/src/modeling/gwlfe/subbasin/templates/sourcesTable.html

```diff
@@ -1 +1,49 @@
-<h3>Source table here</h3>
+<table class="table custom-hover" data-toggle="table">
+    <thead>
+        <tr>
+            <th rowspan="2" class="subbasin-source-col" data-sortable="true" data-sorter="window.numericSort">Sources</th>
+            <th rowspan="2" data-sortable="true" data-sorter="window.numericSort">Area (km<sup>2</sup>)</th>
+            <th colspan="3">Total Loads (not normalized)</th>
+            <th colspan="3">Loading Rates (area normalized)</th>
+        </tr>
+        <tr>
+            <th data-sortable="true" data-sorter="window.numericSort">Sediment (kg/y)</th>
+            <th data-sortable="true" data-sorter="window.numericSort">Total Nitrogen (kg/y)</th>
+            <th data-sortable="true" data-sorter="window.numericSort">Total Phosphorus (kg/y)</th>
+            <th data-sortable="true" data-sorter="window.numericSort">Sediment (kg/ha/y)</th>
+            <th data-sortable="true" data-sorter="window.numericSort">Total Nitrogen (kg/ha/y)</th>
+            <th data-sortable="true" data-sorter="window.numericSort">Total Phosphorus (kg/ha/y)</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for row in rows %}
+            <tr>
+                <td class="text-left">{{ row.Source }}</td>
+                <td class="strong text-right">{{ row.Area|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ row.Sediment|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ row.TotalN|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ row.TotalP|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ (row.Sediment/row.Area)|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ (row.TotalN/row.Area)|round(2)|toLocaleString(2) }}</td>
+                <td class="strong text-right">{{ (row.TotalP/row.Area)|round(2)|toLocaleString(2) }}</td>
+            </tr>
+        {% endfor %}
+    </tbody>
+    <tfoot>
+        <tr>
+            <th class="text-left">{{ summaryRow.Source }}</td>
```

Comment:
Will update this tomorrow to not start with `<th>` and close with `</td>`. Doesn't seem to affect it for testing or anything though.

Reply:
updated!